app/src/main/java/org/mozilla/tv/firefox/pocket/PocketRepoCache.kt

```diff
@@ -28,13 +27,17 @@ open class PocketRepoCache(private val repo: PocketVideoRepo) {
     private val _feedState = MutableLiveData<PocketVideoRepo.FeedState>() // Mutable backer for feedState
     open val feedState: LiveData<PocketVideoRepo.FeedState> = _feedState
 
-    private val observer = Observer<PocketVideoRepo.FeedState> {
-        _feedState.postValue(it)
-    }
-
-    // This should only be called when Pocket videos are not visible to the user.
+    // This should only be unfrozen when Pocket videos are not visible to the user.
     // See class kdoc
-    fun unfreeze() = repo.feedState.observeForever(observer)
+    var frozen = false
```

Comment:
We couldn't add and remove the observer anymore since now we need to let values through based on two conditions instead of one.  

I'm completely agreed that shared mutable state is bad, but personally I don't think freeze/unfreeze is any better.  They're effectively a getter/setter for a mutable boolean, which is a big part of why I was so unhappy about introducing this class in the first place.